src/main/java/tests/DownloadUploadTest.java

```diff
@@ -0,0 +1,46 @@
+package tests;
+
+import configreader.ConfigManager;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.io.File;
+import java.io.IOException;
+
+public class DownloadUploadTest extends BasicTest {
+    private String picturePath = "D:\\java\\projects\\FormAutomatedTest\\src\\main\\resources\\kfc_fighter.jpg";
```

Comment:
oh, very bad practice to put the full path of the file. The file should be in your resources folder and the path should be relative